haproxy/test/controls/command_works.rb

```diff
@@ -0,0 +1,9 @@
+pkg_ident = `grep pkg_ident results/last_build_haproxy.env | cut -d \= -f2`.strip
+installation_directory = "/hab/pkgs/#{pkg_ident}"
+command_test = "#{installation_directory}/bin/haproxy --help 2>&1"
```

Comment:
I think as a standard, the`command_works.rb` should not redirect `stderr` and should assert on `stderr` and `exit_status` e.g.

```
describe bash(command_test) do
    its('stdout') { should match /Usage : haproxy/ }
    its('stderr') { should eq '' }
    its('exit_status') { should cmp 0 }
end
```